src/main/java/com/mateacademy/lessons13/rest/MateGroupServiceImpl.java

```diff
@@ -80,4 +82,77 @@ public Response removeStudent(@PathParam("groupId")int groupId, @PathParam("surn
 
 		return Response.status(ACCEPTED).entity(mateGroup).type(MediaType.APPLICATION_JSON).build();
 	}
+
+	@Override
+	@PUT
+	@Path("/humanResources")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response addHR(@PathParam("groupId")int groupId, HumanResource newHR) {
+
+		MateGroup mateGroup = mateGroups.get(groupId);
+		if (mateGroup != null) {
+			mateGroup.getHumanResources().add(newHR);
+			return Response.status(ACCEPTED).entity(mateGroup).type(MediaType.APPLICATION_JSON).build();
+		}
+		return Response.status(Status.NOT_FOUND).build();
+	}
+
+	@Override
+	@DELETE
+	@Path("/hrs/{surname}")
+	@Produces(MediaType.APPLICATION_JSON)
+	public Response removeHR(@PathParam("groupId")int groupId, @PathParam("surname")String surname) {
+		MateGroup mateGroup = mateGroups.get(groupId);
+		if (mateGroup != null) {
+			mateGroup.getHumanResources().removeIf(f -> f.getSurname().equals(surname));
+			return Response.status(Status.OK).entity(mateGroups.get(groupId).getHumanResources())
```

Comment:
U can use just just mateGroup instead mateGroups.get(groupId).

Reply:
ok

Reply:
"Ok". Where is changes?